extensions/amp-story/0.1/amp-story.js

```diff
@@ -1433,6 +1450,21 @@ export class AmpStory extends AMP.BaseElement {
     this.switchTo_(dev().assertElement(this.pages_[0].element).id);
   }
 
+  /** @return {!NavigationState} */
+  getNavigationState() {
+    return this.navigationState_;
+  }
+
+  /** @return {?./amp-story-page.AmpStoryPage} */
+  getActivePage() {
+    return this.activePage_;
+  }
+
+  /** @return  {!Array<!./amp-story-page.AmpStoryPage>} */
+  getPages() {
+    return this.pages_;
```

Comment:
This in particular makes me nervous, since I think modifying this outside of `AmpStory` could break things unexpectedly (which I understand you're not doing, but this would allow)

Reply:
also probably goes away with `insertPage`

Reply:
@newmuis as we agree in the previous PR, eventually we need a separate PageManager class. 

Do you think we should do it now or wait after AMP conf?